resources/dicts/events/death/death_reactions/general/general_dislike.json

```diff
@@ -6,7 +6,11 @@
       "r_c wanders past m_c's vigil. {PRONOUN/r_c/subject/CAP} won't mourn m_c's passing.",
       "Frankly, they're all better off without m_c, and r_c doesn't know why everyone else is so grief stricken.",
       "As much as {PRONOUN/r_c/subject} hated m_c, the sight of {PRONOUN/m_c/poss} body still fills r_c with dread.",
-      "Yes, {PRONOUN/r_c/subject} hated m_c, but m_c was {PRONOUN/r_c/poss} Clanmate first and foremost, and r_c bows {PRONOUN/r_c/poss} head at the vigil respectfully."
+      "Yes, {PRONOUN/r_c/subject} hated m_c, but m_c was {PRONOUN/r_c/poss} Clanmate first and foremost, and r_c bows {PRONOUN/r_c/poss} head at the vigil respectfully.",
+      "With m_c sadly gone from this world, r_c didn't take much to openly mock the decease cat's flaws and mistakes for the Clan. With them gone, the Clan can run perfectly now.",
+      "Even with m_c's family and friends at the vigil, r_c bodly claimed {PRONOUN/m_c/subject} weren't cut out to be part of this Clan.",
```

Comment:
"bodly" should be "boldly".  "claimed" should be "claims", "weren't" should be "wasn't" and the last pronoun should probably be replaced with m_c's name to avoid they/them pronoun confusion.